### DISCUSSION TITLE
Help about wifi

### DISCUSSION THREAD

**Discussion by ddcorazon** [roleoroleo/yi-hack-Allwinner-v2#890] [Q&A]
Hey @roleoroleo,

Thanks for your job and i hope you're doing well

I had an issue last night with my Yi Dome Camera U (2K) on which I had installed the hack. It kept saying repeatedly that it was waiting to connect to Wi-Fi (as if the hack was disabled), and this morning after several unsuccessful reboots (both the camera and my access point), I had to reset it and connected via the Yi home app before managing to reconnect the hack

For context, my cameras at home do not have internet access (internet is intentionally blocked), so my question is whether that could be the cause, knowing that I used the hack without internet for 5 days without any issues.

Also, how does the camera verify that it is connected to Wi-Fi? Is it via a ping?

**Comment by roleoroleo** [comment]
Pairing the cam with the app is the better solution to the wifi connection problem.

> For context, my cameras at home do not have internet access (internet is intentionally blocked), so my question is whether that could be the cause, knowing that I used the hack without internet for 5 days without any issues.

I don't know if this could be the cause. Other users noticed some problems with firewall blocked cams.
But I wasn't able to fix the problem.
The reason is because wifi connection is handled by yi processes and I can't change this behavior with the hack.
